src/mixer.rs

```diff
@@ -210,11 +201,12 @@ pub fn load_samples_from_file(bytes: &[u8]) -> Result<Vec<[f32; 2]>, ()> {
     if sample_rate != 44100 {
         let new_length = ((44100 as f32 / sample_rate as f32) * frames.len() as f32) as usize;
 
-        let mut resampled = vec![[0.0; 2]; new_length];
+        let mut resampled = vec![0.0; new_length];
```

Comment:
shouldn't it be `new_length * 2`? 
Looks like without `* 2` fewer floats will end up in `resampled` vec?

Reply:
but it will play twice faster than(with a sample rate of 22500), so if it works - probable its all right and I just miss something :)

Reply:
It's using `frames` as a base length which is already "doubled" since it's also using `f32` instead of `[f32; 2]`.

Reply:
ohh missed this `frames.len()`

Reply:
I think there might be an issue here after all: if `new_length` ends up being an odd number line 209 is going to panic when accessing `sample[1]`.